content/docs/guide/gamemodes/sticky_jump.md

```diff
@@ -10,18 +10,16 @@ tags:
 
 ## What is Sticky Jump?
 
-Based on the Demoman class from Team Fortress 2, players shoot explosives that stick to surfaces and have control over their detonation which propels themselves throughout maps. Up to 3 of these arching bombs can be used at once to go at extremely high speeds, or they can be in alternation to pogo on the floor, the wall, or even in the air.
-
-Sticky Jump movement can be found in Team Fortress 2, however due to the high damage that stickybombs deal, it has extremely limited use in competitive games. Instead, most sticky jumping is done on special 'jump' maps and servers that give the player infinite health and ammo.
-
-The version of Sticky Jump implemented in Momentum Mod has a few key changes from Team Fortress 2. Most notably, all health and ammo is gone. Instead, players are limited to having 3 stickybombs out at any given time and further restrictions are done using special triggers in the maps.
+Based on the Demoman class from Team Fortress 2, players shoot explosives that stick to surfaces and have control over their detonation which propels themselves throughout maps. Up to 3 of these arcing bombs can be used at once to go at extremely high speeds, or they can be in alternation to pogo on the floor, the wall, or even in the air.
```

Comment:
Might want to give a tiny definition to what "pogo" is in this sentence, maybe along the lines of "or they can be used in alternation to bounce repeatedly off the floors, walls, or the air itself without touching the ground (also called pogo)"

Reply:
"Pogo" is kind of self-explanatory, no? The rocket jump guide also doesn't define pogo, either. Although, maybe I could put pogo in quotes in the actual description to denote that pogo is the actual name, instead of _just_ a descriptor.